prov/kubernetes/execution.go

```diff
@@ -398,6 +412,51 @@ func (e *execution) manageServiceResource(ctx context.Context, clientset kuberne
 	return nil
 }
 
+//Manage kubernetes PersistentVolumeClaim
+func (e *execution) manageSimpleResourcePVC(ctx context.Context, clientset kubernetes.Interface, generator *k8sGenerator, operationType k8sResourceOperation, rSpec string) (err error) {
+	if rSpec == "" {
+		return errors.Errorf("Missing mandatory resource_spec property for node %s", e.nodeName)
+	}
+	var pvcRepr apiv1.PersistentVolumeClaim
+	//Unmarshall
```

Comment:
comment redundent with code it comments (.Unmarshall)

Reply:
Solved